src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -895,6 +898,10 @@ export default {
 			if (!this.isTributePickerActive) {
 				this.blurInput()
 			}
+		},
+
+		isMobile() {
+			return /Android|iPhone|iPad|iPod/i.test(navigator.userAgent)
```

Comment:
Should we use the `isMobile` mixing from the vue lib instead?
https://github.com/nextcloud/nextcloud-vue/blob/master/src/utils/IsMobileState.js
But seems to go by resolution rather then user agent.

Reply:
Yes, in this case here I would by purpose use the user agent instead of the resolution since the size of the window could also be small on desktop and you would still want to focus the input for them. This is also how it was done in https://github.com/nextcloud/spreed/pull/8058/files